server/model/environments.go

```diff
@@ -0,0 +1,47 @@
+package model
+
+import (
+	"strings"
+	"time"
+)
+
+type (
+	Environment struct {
+		ID          string
+		Name        string
+		Description string
+		CreatedAt   time.Time
+		Values      []EnvironmentValue
+	}
+
+	EnvironmentValue struct {
+		Key   string
+		Value string
+	}
+)
+
+func (e Environment) GetSlug() string {
+	return strings.ToLower(strings.ReplaceAll(strings.TrimSpace(e.Name), " ", "-"))
+}
+
+func (e Environment) Merge(env Environment) Environment {
+	values := make(map[string]string)
```

Comment:
I guess the idea of this is to avoid duplicated env keys. but in doing this, it's loosing the order guaranteed in a slice, which I imagine is the point of `e.Values` being a slice.

The only way to keep order and uniqueness of a map is using `OrderedMap`, even if the code looks a bit to verbose

Reply:
yeah, the order doesn't matter in this case. I'll change it to `map[string]string` in a different PR